apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1764,6 +1764,14 @@ public static Bundle createBundle(
        * TODO: Above bug should be fixed as of 1/1/20; re-investigate this
        */
       int endIndex = Math.min(paging.getStartIndex() + paging.getPageSize(), resources.size());
+      // Throw a 400 if startIndex >= results, since we cant sublist with these values
+      if (paging.getStartIndex() >= resources.size()) {
+        throw new InvalidRequestException(
+            String.format(
+                "Value for startIndex (%d) must be less than than result size (%d)",
+                paging.getStartIndex(), resources.size()));
+      }
```

Comment:
Might be a good candidate for a utility method.  Seeing exactly the same lines of code in 3 different classes.

Reply:
I'll make one for the TransformerUtilsV2 since it gets shared between here and PAC code, but two usages are in the separate transformerUtils versions which dont have a good place to share code between them. Maybe in future work to pull out shared code we can reuse the new util method I'll make in both v1 and v2 locations and put it in some commonTransformerUtils class or something.